CHANGELOG.md

```diff
@@ -1,3 +1,268 @@
+| "/wd/write"                              | []StatusItem        | fsi.Write                        |
+| "/wd/createlink"                         | VersionInfo         | fsi.CreateLink                   |
+| "/wd/unlink"                             | string              | fsi.Unlink                       |
+| "/wd/ensureref"                          | --                  | fsi.EnsureRefNotLinked           |
+|                                          |                     |                                  |
+
+### "Protocol Endpoints"
+
+| Sync Endpoints                           |                     |                  |                                  |
+| ---------------------------------------- | ------------------- | ---------------- | -------------------------------- |
+| "/sync/dag"                              | GET *1              | dag.Info         | remote.DsyncHTTPHandler          |
+| "/sync/dag"                              | GET *2              | []byte           | remote.DsyncHTTPHandler          |
+| "/sync/dag"                              | POST                | dag.Manifest     | remote.DsyncHTTPHandler          |
+| "/sync/dag"                              | PUT                 | --               | remote.DsyncHTTPHandler          |
+| "/sync/dag"                              | PATCH               | stream?          | remote.DsyncHTTPHandler          |
+| "/sync/logs"                             | GET                 | --               | remote.LogbookHTTPHandlery       |
+| "/sync/logs"                             | PUT                 | []byte           | remote.LogbookHTTPHandlery       |
+| "/sync/logs"                             | DELETE              | --               | remote.LogbookHTTPHandlery       |
+| "/sync/resolve"                          | GET                 | DSRef            | remote.RefsHTTPHandler           |
+| "/sync/resolve"                          | DELETE              | --               | remote.RefsHTTPHandler           |
+|                                          |                     |                  |                                  |
+
+
+## Redesigned lib interface
+In general, we've streamlined the core functionality and reconciled input params in the `lib` package (which contain the methods and params that power both the api and cmd), so that no matter how you are accessing functionality in qri, whether using `lib` as a package,  using the api, or using the command line, you will have, not only consistent input parameters, but also consistent behavior. We're also utilizing our new `dispatch` pattern to replace our old rpc client with the same http json api that we expose to our users. That way all our api's, http or otherwise, have the same expectations. If something is broke in one place, it is broken in all places, consequently, when it is fixed in one place it will be fixed in all places.
```

Comment:
```suggestion
In general, we've streamlined the core functionality and reconciled input params in the `lib` package (which contain the methods and params that power both the api and cmd), so that no matter how you are accessing functionality in qri, whether using `lib` as a package,  using the HTTP API, or using the command line, you can expect consistent inputs and–more importantly–consistent behavior. We're also utilizing our new `dispatch` pattern to replace our old rpc client with the _same_ JSON HTTP API exposed to users. That way all our API's, HTTP or otherwise have the same expectations. If something is broke in one place, it is broken in all places, consequently, when it is fixed in one place it will be fixed in all places!
```